### PR TITLE
Assignees

### PR DESCRIPTION
**Pull request by maxice8**
The data used to find those out is from `gitlab.alpinelinux.org` (GitLab Community Edition 13.9.7)

```json
  "assignees": [
    {
      "id": 1589,
      "name": "Leo",
      "username": "Leo",
      "avatar_url": "https://gitlab.alpinelinux.org/uploads/-/system/user/avatar/1589/avatar.png",
      "email": "meme"
   }
  ]
```

### CODE REVIEW

**Comment by maxice8** [comment]
Seems like `User1` appears enough the linter wants it `const`

File: event_webhook_types.go [hunk 1]
```diff
@@ -536,9 +536,10 @@ type MergeEvent struct {
 		OldRev         string        `json:"oldrev"`
 		Assignee       MergeAssignee `json:"assignee"`
 	} `json:"object_attributes"`
-	Repository *Repository   `json:"repository"`
-	Assignee   MergeAssignee `json:"assignee"`
-	Labels     []Label       `json:"labels"`
+	Repository *Repository     `json:"repository"`
+	Assignee   MergeAssignee   `json:"assignee"`
+	Assignees  []MergeAssignee `json:"assignees"`
```
Comment (svanharmelen) [inline event_webhook_types.go:541]:
```suggestion
	Assignees  []*MergeAssignee `json:"assignees"`
```

**Review by svanharmelen — CHANGES_REQUESTED** [review]
One small comment, and guess you need to fix the linter comment as well 😞

**Comment by svanharmelen** [comment]
If you rebase the linter should not complain anymore 🤞🏻

File: event_webhook_types.go (hunk 1 above)
Comment (maxice8) [reply event_webhook_types.go]: Thanks, applied

**Comment by maxice8** [comment]
Made them constants.

**Review by svanharmelen — APPROVED** [review]
Thanks @maxice8 👍🏻

**Pull request merged**